internal/ethapi/api.go

```diff
@@ -141,6 +141,36 @@ func (s *PublicTxPoolAPI) Content() map[string]map[string]map[string]*RPCTransac
 	return content
 }
 
+// ContentFrom returns the transactions contained within the transaction pool.
+func (s *PublicTxPoolAPI) ContentFrom(addr common.Address) map[string]map[string]*RPCTransaction {
+	content := map[string]map[string]*RPCTransaction{
+		"pending": make(map[string]*RPCTransaction),
+		"queued":  make(map[string]*RPCTransaction),
+	}
+	pending, queue := s.b.TxPoolContent()
+	curHeader := s.b.CurrentHeader()
+	// Build the pending transactions
+	txs, ok := pending[addr]
+	if ok {
+		dump := make(map[string]*RPCTransaction)
+		for _, tx := range txs {
+			dump[fmt.Sprintf("%d", tx.Nonce())] = newRPCPendingTransaction(tx, curHeader, s.b.ChainConfig())
+		}
+		content["pending"] = dump
+	}
```

Comment:
Nitpick:
```suggestion
	if txs, ok := pending[addr]; ok {
		dump := make(map[string]*RPCTransaction)
		for _, tx := range txs {
			dump[fmt.Sprintf("%d", tx.Nonce())] = newRPCPendingTransaction(tx, curHeader, s.b.ChainConfig())
		}
		content["pending"] = dump
	}
```
And the same below